types/database/models/servers/user.ts

```diff
@@ -8,6 +8,7 @@ import type PreferenceModel from './preference';
 import type ReactionModel from './reaction';
 import type TeamMembershipModel from './team_membership';
 import type ThreadParticipantsModel from './thread_participant';
+import type CustomProfileAttributeModel from '@database/models/server/custom_profile_attribute';
```

Comment:
I know it is weird, but we have two definitions of the models. One under the `types` folder, and another one in the `models` folder. You are missing the one in the types folder, and this should point to that one.